### ISSUE TITLE
Fail to load config file on Python 3.10

### ISSUE THREAD

**Issue by martadinata666** [nicolargo/glances#2176]
**Describe the bug**
Fail to load config file glances 3.3.0 with python 3.10

**To Reproduce**
1. Install glances 3.3.0
2. Load glances with command `glances --disable-autodiscover -C /home/debian/glances/glances.conf --browser`
3. Glances throw err 
```
Traceback (most recent call last):
  File "/usr/local/bin/glances", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.10/dist-packages/glances/__init__.py", line 179, in main
    start(config=core.get_config(), args=core.get_args())
  File "/usr/local/lib/python3.10/dist-packages/glances/__init__.py", line 108, in start
    mode = GlancesMode(config=config, args=args)
  File "/usr/local/lib/python3.10/dist-packages/glances/client_browser.py", line 37, in __init__
    self.load()
  File "/usr/local/lib/python3.10/dist-packages/glances/client_browser.py", line 54, in load
    self.password = GlancesPassword(config=self.config)
  File "/usr/local/lib/python3.10/dist-packages/glances/password_list.py", line 23, in __init__
    super(GlancesPasswordList, self).__init__()
  File "/usr/local/lib/python3.10/dist-packages/glances/password.py", line 33, in __init__
    self.password_dir = self.local_password_path()
  File "/usr/local/lib/python3.10/dist-packages/glances/password.py", line 41, in local_password_path
    return self.config.get_value('passwords',
AttributeError: 'NoneType' object has no attribute 'get_value'
```

**Expected behavior**
Load list of monitored server

**Desktop (please complete the following information):**
 - OS: Debian Sid
 - Glances Version 3.3.0
 - Glances logs file [output of tail -10 <glances logs file>]
 - Glances test [output of glances --issue] (only available with Glances 3.1.7 or higher)
 ```
debian@4aa997dcbc06:~$ glances --issue
=====================================================================================================================================================
Glances 3.3.0 (/usr/local/lib/python3.10/dist-packages/glances/__init__.py)
Python 3.10.8 (/usr/bin/python3)
PsUtil 5.9.3 (/usr/local/lib/python3.10/dist-packages/psutil/__init__.py)
=====================================================================================================================================================
alert         [OK]    0.00001s []
amps          [OK]    0.00005s []
cloud         [OK]    0.00003s {}
connections   [OK]    0.00319s {'LISTEN': 3, 'ESTABLISHED': 1, 'SYN_SENT': 0, 'SYN_RECV': 0, 'initiated': 0, 'terminated': 0, 'nf_conntrack
core          [OK]    0.00028s {'phys': 6, 'log': 6}
cpu           [OK]    0.00021s {'total': 39.8, 'user': 25.3, 'nice': 0.0, 'system': 6.4, 'idle': 58.2, 'iowait': 9.5, 'irq': 0.0, 'softirq'
diskio        [OK]    0.00055s key=disk_name [{'time_since_update': 2.022228479385376, 'disk_name': 'loop0', 'read_count': 0, 'write_count'
docker        [OK]    0.00003s []
folders       [OK]    0.00002s []
fs            [OK]    0.00044s key=mnt_point [{'device_name': '/dev/sdb2', 'fs_type': 'ext4', 'mnt_point': '/etc/resolv.conf', 'size': 1170
gpu           [OK]    0.00002s []
help          [OK]    0.00000s None
ip            [OK]    0.00020s {'gateway': '172.19.0.1', 'address': '172.19.0.46', 'mask': '255.255.0.0', 'mask_cidr': 16, 'public_address'
irq           [OK]    0.00038s key=irq_line [{'irq_line': 'LOC', 'irq_rate': 24665, 'key': 'irq_line', 'time_since_update': 2.0445432662963
load          [OK]    0.00003s {'min1': 3.69970703125, 'min5': 3.90478515625, 'min15': 5.087890625, 'cpucore': 6}
mem           [OK]    0.00013s {'total': 33307971584, 'available': 13723262976, 'percent': 58.8, 'used': 19584708608, 'free': 13723262976, 
memswap       [OK]    0.00015s {'total': 16653983744, 'used': 8839081984, 'free': 7814901760, 'percent': 53.1, 'sin': 69139730432, 'sout': 
network       [OK]    0.00028s key=interface_name [{'interface_name': 'lo', 'alias': None, 'time_since_update': 2.007660388946533, 'cumulat
now           [OK]    0.00001s 2022-10-30 02:44:04 WIB
percpu        [OK]    0.00006s key=cpu_number [{'key': 'cpu_number', 'cpu_number': 0, 'total': 36.1, 'user': 22.2, 'system': 4.6, 'idle': 6
ports         [OK]    0.00018s []
processcount  [OK]    0.00148s {'total': 6, 'running': 2, 'sleeping': 4, 'thread': 6, 'pid_max': 0}
processlist   [OK]    0.00003s key=pid [{'pid': 64, 'name': 'glances', 'cpu_times': pcputimes(user=1.39, system=0.06, children_user=0.0, ch
psutilversion [OK]    0.00002s (5, 9, 3)
quicklook     [OK]    0.00025s {'cpu': 39.8, 'percpu': [{'key': 'cpu_number', 'cpu_number': 0, 'total': 36.1, 'user': 22.2, 'system': 4.6, 
raid          [OK]    0.00010s {}
sensors       [OK]    0.00001s key=label [{'label': 'Composite', 'unit': 'C', 'value': 47, 'warning': 79, 'critical': 84, 'type': 'temperat
smart         [NA]
system        [OK]    0.00001s {'os_name': 'Linux', 'hostname': '4aa997dcbc06', 'platform': '64bit', 'linux_distro': 'Debian GNU/Linux', 'o
uptime        [OK]    0.00008s {'seconds': 668655}
wifi          [OK]    0.00002s []
=====================================================================================================================================================
Total time to update all stats: 0.00919s
=====================================================================================================================================================
 ```

**Additional context**
As 3.3.0 installed on python 3.10, cant really tell if glances 3.2.7 will behave the same. But i can confirm that glances 3.2.7 work correctly with python 3.9.

** Config file **
```
##############################################################################
# Client/server
##############################################################################

[serverlist]
# Define the static servers list
server_1_name=192.168.0.2
server_1_alias=homelab
server_1_port=61200

server_2_name=192.168.0.28
server_2_alias=gitlab
server_2_port=61201

server_3_name=192.168.0.15 
server_3_alias=rpi4 
server_3_port=61202

#[passwords]
# Define the passwords list
# Syntax: host=password
# Where: host is the hostname
#        password is the clear password
# Additionally (and optionally) a default password could be defined
#localhost=abc
```

**Comment by nicolargo** [comment]
Please update to 3.3.0.1
It should solve this issue.

**Comment by martadinata666** [comment]
oh i see, i just follow release from github, and don't realize there is 3.3.0.1, thanks. that solved it

**Comment by martadinata666** [comment]
More info glances 3.3.0.1

```
debian@de1d2df9448c:~$ glances --version
Glances v3.3.0.1 with PsUtil v5.9.3
Log file: /home/debian/.local/share/glances/glances.log
```

Run command: `glances --disable-autodiscover -C /home/debian/glances/glances.conf --browser`
```
Traceback (most recent call last):
  File "/usr/local/bin/glances", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.10/dist-packages/glances/__init__.py", line 179, in main
    start(config=core.get_config(), args=core.get_args())
  File "/usr/local/lib/python3.10/dist-packages/glances/__init__.py", line 108, in start
    mode = GlancesMode(config=config, args=args)
  File "/usr/local/lib/python3.10/dist-packages/glances/client_browser.py", line 37, in __init__
    self.load()
  File "/usr/local/lib/python3.10/dist-packages/glances/client_browser.py", line 54, in load
    self.password = GlancesPassword(config=self.config)
  File "/usr/local/lib/python3.10/dist-packages/glances/password_list.py", line 23, in __init__
    super(GlancesPasswordList, self).__init__()
  File "/usr/local/lib/python3.10/dist-packages/glances/password.py", line 33, in __init__
    self.password_dir = self.local_password_path()
  File "/usr/local/lib/python3.10/dist-packages/glances/password.py", line 41, in local_password_path
    return self.config.get_value('passwords',
AttributeError: 'NoneType' object has no attribute 'get_value'
```

glances --issue
```
==================================================================================================================================================================================================================================
Glances 3.3.0.1 (/usr/local/lib/python3.10/dist-packages/glances/__init__.py)
Python 3.10.8 (/usr/bin/python3)
PsUtil 5.9.3 (/usr/local/lib/python3.10/dist-packages/psutil/__init__.py)
==================================================================================================================================================================================================================================
alert         [OK]    0.00001s []
amps          [OK]    0.00006s []
cloud         [OK]    0.00003s {}
connections   [OK]    0.00434s {'LISTEN': 3, 'ESTABLISHED': 1, 'SYN_SENT': 0, 'SYN_RECV': 0, 'initiated': 0, 'terminated': 0, 'nf_conntrack_count': 14.0, 'nf_conntrack_max': 262144.0, 'nf_conntrack_percent': 0.005340
core          [OK]    0.00045s {'phys': 6, 'log': 6}
cpu           [OK]    0.00029s {'total': 71.4, 'user': 26.8, 'nice': 0.0, 'system': 7.5, 'idle': 63.6, 'iowait': 1.4, 'irq': 0.0, 'softirq': 0.7, 'steal': 0.0, 'guest': 1.9, 'guest_nice': 0.0, 'time_since_update': 2.
diskio        [OK]    0.00080s key=disk_name [{'time_since_update': 2.02602219581604, 'disk_name': 'loop0', 'read_count': 0, 'write_count': 0, 'read_bytes': 0, 'write_bytes': 0, 'key': 'disk_name'}, {'time_since_upda
docker        [OK]    0.00004s []
folders       [OK]    0.00003s []
fs            [OK]    0.00062s key=mnt_point [{'device_name': '/dev/sdb2', 'fs_type': 'ext4', 'mnt_point': '/etc/resolv.conf', 'size': 117019152384, 'used': 62187257856, 'free': 48840331264, 'percent': 56.0, 'key': '
gpu           [OK]    0.00003s []
help          [OK]    0.00000s None
ip            [OK]    0.00026s {'gateway': '172.19.0.1', 'address': '172.19.0.46', 'mask': '255.255.0.0', 'mask_cidr': 16, 'public_address': '175.158.53.251', 'public_info_human': ''}
irq           [OK]    0.00053s key=irq_line [{'irq_line': 'LOC', 'irq_rate': 25541, 'key': 'irq_line', 'time_since_update': 2.045518398284912}, {'irq_line': 'CAL', 'irq_rate': 4175, 'key': 'irq_line', 'time_since_upd
load          [OK]    0.00005s {'min1': 3.0888671875, 'min5': 3.61328125, 'min15': 4.24267578125, 'cpucore': 6}
mem           [OK]    0.00016s {'total': 33307971584, 'available': 13983862784, 'percent': 58.0, 'used': 19324108800, 'free': 13983862784, 'active': 10377764864, 'inactive': 13347356672, 'buffers': 793579520, 'cached
memswap       [OK]    0.00022s {'total': 16653983744, 'used': 9889116160, 'free': 6764867584, 'percent': 59.4, 'sin': 70436626432, 'sout': 118627749888, 'time_since_update': 2.991398811340332}
network       [OK]    0.00039s key=interface_name [{'interface_name': 'lo', 'alias': None, 'time_since_update': 2.0105602741241455, 'cumulative_rx': 3843, 'rx': 0, 'cumulative_tx': 3843, 'tx': 0, 'cumulative_cx': 768
now           [OK]    0.00001s 2022-10-30 09:39:45 WIB
percpu        [OK]    0.00008s key=cpu_number [{'key': 'cpu_number', 'cpu_number': 0, 'total': 66.7, 'user': 43.6, 'system': 19.7, 'idle': 33.3, 'nice': 0.0, 'iowait': 3.4, 'irq': 0.0, 'softirq': 0.0, 'steal': 0.0, '
ports         [OK]    0.00025s []
processcount  [OK]    0.00213s {'total': 6, 'running': 1, 'sleeping': 5, 'thread': 6, 'pid_max': 0}
processlist   [OK]    0.00003s key=pid [{'memory_info': pmem(rss=40026112, vms=347164672, shared=12763136, text=2768896, lib=0, data=61976576, dirty=0), 'pid': 77, 'memory_percent': 0.12016976746559722, 'nice': 0, 'n
psutilversion [OK]    0.00003s (5, 9, 3)
quicklook     [OK]    0.00035s {'cpu': 71.4, 'percpu': [{'key': 'cpu_number', 'cpu_number': 0, 'total': 66.7, 'user': 43.6, 'system': 19.7, 'idle': 33.3, 'nice': 0.0, 'iowait': 3.4, 'irq': 0.0, 'softirq': 0.0, 'steal
raid          [OK]    0.00013s {}
sensors       [OK]    0.00001s key=label [{'label': 'Composite', 'unit': 'C', 'value': 46, 'warning': 79, 'critical': 84, 'type': 'temperature_core', 'key': 'label'}, {'label': 'SYSTIN', 'unit': 'C', 'value': 113, 'w
smart         [NA]
system        [OK]    0.00001s {'os_name': 'Linux', 'hostname': 'de1d2df9448c', 'platform': '64bit', 'linux_distro': 'Debian GNU/Linux', 'os_version': '5.15.0-52-lowlatency', 'hr_name': 'Debian GNU/Linux 64bit'}
uptime        [OK]    0.00012s {'seconds': 693596}
wifi          [OK]    0.00003s []
==================================================================================================================================================================================================================================
Total time to update all stats: 0.01290s
==================================================================================================================================================================================================================================
```

Is there something to do with `password`?

**Comment by nicolargo** [comment]
Just publish an hard mpatch on the master branch for this issue 'also including https://github.com/nicolargo/glances/issues/2143

Please upgrade to Glances version 3.3.0.2.

Thanks !
